oxidation/libparsec/crates/tests_fixtures/Cargo.toml

```diff
@@ -5,8 +5,9 @@ edition = "2021"
 
 [dependencies]
 libparsec_crypto = { path = "../crypto" }
```

Comment:
Do we still need this dependency ? (I suspect removing it won't change the compilation time given types itself need it, but it simplify it little bit ^^)

Reply:
No, we don't need it. Removed now